Ginger/GingerCoreNET/Run/RunListenerLib/SealightsExecutionLogger/SealightsReportExecutionLogger.cs

```diff
@@ -253,14 +269,6 @@ public override async void ActionEnd(uint eventTime, Act action, bool offlineMod
         }
 
 
-        #endregion Action  
```

Comment:
why this function was removed?

Reply:
not sure why it is shown s removed, but this is still exists on the file